support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -159,14 +173,19 @@ export function ContributionsOrderSummary({
 }: ContributionsOrderSummaryProps): JSX.Element {
 	const [showCheckList, setCheckList] = useState(false);
 
-	const hasCheckList = enableCheckList && checkListData.length > 0;
+	const hasCheckList =
+		enableCheckList && (checkListData.length > 0 || !!productDescription);
+
 	const checkList = hasCheckList && (
 		<CheckList
-			checkListData={checkListData}
+			checkListData={
+				checkListFromBenefits(productDescription?.benefits) ?? checkListData
+			}
 			style="compact"
 			iconColor={palette.brand[500]}
 		/>
 	);
+	const productName = productDescription?.label ?? description;
```

Comment:
I'm not sure if this is needed given that where we actually get the data from in `checkout`.

Reply:
Agree removed